### DISCUSSION TITLE
Quarto v1.4 Inline Execution

### DISCUSSION THREAD

**Discussion by pedromsouza** [quarto-dev/quarto-cli#7121] [Q&A]
### Description

I'm trying to use Inline Execution for to show the value of a python variable in the markdown section, but it's not working. 

https://quarto.org/docs/prerelease/1.4/inline.html

Example:
````{python}
x = 5
````

The answer is `{python} x`

My Code:

````{python}
v2022 = int(df4[(df4['Código'] == 1268) & (df4['ANO'] == 2022)].groupby('ANO')['PROCESSO'].sum())
v2019 = int(df4[(df4['Código'] == 1268) & (df4['ANO'] == 2019)].groupby('ANO')['PROCESSO'].sum())
var = ((v2022-v2019)/v2019*100)````

A variação entre 2019 e 2022 foi de: `{python} var`

**Comment by mcanouil** [comment]
Could you please fix or format your example properly as currently backticks are not matched.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Reply by mcanouil** [reply]
@pedromsouza please share something exactly like the example I shared. Without it's hard to understand exactly what's your issue/use case. Thank you.

**Comment by pedromsouza** [comment]
[div.zip](https://github.com/quarto-dev/quarto-cli/files/12807199/div.zip)

This is the full script. I was unable to paste it all in here.

**Reply by cscheid** [reply]
Hi - can you try to provide a smaller, ideally minimal example? Your .ipynb contains a large number of other things, including the fact that it's loading a file that we don't have access to.
